src/components/AssetsTableContent.jsx

```diff
@@ -73,7 +73,7 @@ const AssetsTableContent = (props) => {
             return (
               <TableRowDetail
                 viewDetailsRoute={assetViewUrl}
-                key={asset.id}
+                key={asset.id || asset.uuid}
```

Comment:
what is the difference between the id and uuid? In which case would you have one and not the other

Reply:
`id` is the unique identifier when Assets are returned from the Asset Component using the `assets` endpoint while `uuid` id the unique identifier when Assets are returned from assets assigned to a department.